wasmtime-wasi/Cargo.toml

```diff
@@ -13,18 +13,12 @@ readme = "README.md"
 wasmtime-runtime = { path = "../wasmtime-runtime" }
 wasmtime-environ = { path = "../wasmtime-environ" }
 wasmtime-jit = { path = "../wasmtime-jit" }
+wasi-common = { git = "https://github.com/CraneStation/wasi-common", branch="missing-hostcalls" }
```

Comment:
When we merge `missing-hostcalls` into `master` in `wasi-common`, I'll update this line to reflect this.

Reply:
Updated to track master in `wasi-common`.